### ISSUE TITLE
Sponge messing up our creeper protection

### ISSUE THREAD

**Issue by Raycoms** [SpongePowered/SpongeForge#1935]
**I am currently running**

- SpongeForge version: 1.12.2-2555-7.1.0-BETA-2821
- Forge version: 2555
- Java version:  1.8
- Operating System: Debian

- Plugins/Mods:
 We tested it with and without other mods (besides ours) and it still happends.

Description:
Our mod offers an intensive anti griefing protection for each colony to avoid mob and player griefing of the colonies.
As you'll see in the handler under the description we cancel all explosion events within the colony which works pefectly on any server and singleplayer with older sponge versions or without.
As does it work with this sponge version:
 spongeforge-1.12.2-2529-7.0.0-BETA-2744.jar
The handler:
https://github.com/Minecolonies/minecolonies/blob/version/1.12/src/main/java/com/minecolonies/coremod/permissions/ColonyPermissionEventHandler.java

This bug is very very bad since it destroys several buildings which are crucial for our mod.
This bug persists since 1.10 and has not been resolved but several times been reported yet.

- Logs
Nothing happends in the log which could explain this problem.

**Comment by ryantheleach** [comment]
This will likely be fixed by: https://github.com/SpongePowered/SpongeForge/pull/1911

**Comment by ryantheleach** [comment]
Closing as a duplicate of: https://github.com/SpongePowered/SpongeForge/issues/1700

If this issue remains after #1911 is merged, feel free to create a new ticket.
